e2e/gui/src/test/java/com/epam/pipeline/autotests/utils/Utils.java

```diff
@@ -360,4 +361,8 @@ public static void restartBrowser(final String address) {
         Selenide.close();
         Selenide.open(address);
     }
+
+    public static void click(final By selector) {
+        $(selector).shouldBe(visible).click();
```

Comment:
You can implement `AccessObject` interface if you need to use safe click. Probably, there is no need for `Utils.click`.

Reply:
@tcibinan It's impossible implement `AccessObject` for `Navigation` interface.